src/hv_anndata/manifoldmap.py

```diff
@@ -311,17 +311,19 @@ def _apply_categorical_datashading(
     aggregator = ds.count_cat(color_by)
     # Selector used as a workaround to display categorical counts per pixel
     # One day done directly in Bokeh, see https://github.com/bokeh/bokeh/issues/13354
-    selector = ds.first(plot.kdims[0].name)
-    plot = hd.datashade(plot, aggregator=aggregator, selector=selector, color_key=cmap)
+    plot = hd.datashade(
+        plot,
+        aggregator=aggregator,
+        selector=ds.first(),
+        color_key=cmap,
+    )
     plot = hd.dynspread(plot, threshold=0.5)
     return plot.opts(
         tools=[
             "hover",
             BoxSelectTool(persistent=True),
             LassoSelectTool(persistent=True),
         ],
-        # Override hover_tooltips to exclude the selector value
-        hover_tooltips=[("Label", str(color_by))],
```

Comment:
Why remove it?

Reply:
You're right. Somehow missed that as the dots are quite tiny and I saw the value being `-`. Reverted.

Reply:
Thankfully the `-` entries will be hidden soon.